src/domain/entities/testing/mocks.ts

```diff
@@ -149,8 +149,18 @@ export const generateConnectInstallation = ({
 export const generateAtlassianDesign = ({
 	id = `${generateFigmaFileKey()}/${generateFigmaNodeId()}`,
 	displayName = `Design ${uuidv4()}`,
-	url = `https://www.figma.com/file/UcmoEBi9SyNOX3SNhXqShY/${displayName}?type=design&node-id=0-1&mode=design`,
-	liveEmbedUrl = `https://www.figma.com/file/UcmoEBi9SyNOX3SNhXqShY/${displayName}?type=design&node-id=0-1&mode=design`,
+	url = generateFigmaDesignUrl({
+		fileKey: getFileKeyFromId(id),
+		nodeId: getNodeIdFromId(id),
+		fileName: displayName,
+		mode: 'design',
+	}),
+	liveEmbedUrl = generateFigmaDesignUrl({
+		fileKey: getFileKeyFromId(id),
```

Comment:
**nitpick** Consider using the existing API to get Figma `fileKey` and `nodeId` from `AtlassianDesign.id`.

```ts
url = generateFigmaDesignUrl({
		fileKey: FigmaDesignIdentifier.fromAtlassianDesignId(id).fileKey,
		nodeId: FigmaDesignIdentifier.fromAtlassianDesignId(id).nodeId,
    // ...
})
```

As a result, we could remove `getNodeIdFromId` and `getFileKeyFromId` from this file :)

Reply:
ah nice!